server/jamovi/core.pyx

```diff
@@ -257,18 +266,21 @@ cdef class Column:
         else:
             self._this.append[int](value)
 
-    def append_level(self, raw, label):
-        self._this.appendLevel(raw, label.encode('utf-8'))
+    def append_level(self, raw, label, importValue):
+        self._this.appendLevel(raw, label.encode('utf-8'), importValue.encode('utf-8'))
 
-    def insert_level(self, raw, label):
-        self._this.insertLevel(raw, label.encode('utf-8'))
+    def insert_level(self, raw, label, importValue):
+        self._this.insertLevel(raw, label.encode('utf-8'), importValue.encode('utf-8'))
 
     def get_value_for_label(self, label):
         return self._this.valueForLabel(label.encode('utf-8'))
 
     def clear_levels(self):
         self._this.clearLevels()
 
+    def update_level_counts(self):
```

Comment:
is this guy only used internally? start him with an underscore: `def _update_level_counts()`